### PR TITLE
LPS-177233 - Internal error on Entries when Formula Build has incorrect fields

### PR DESCRIPTION
**Pull request by Artartur**
Related Issue: [LPS-177233](https://issues.liferay.com/browse/LPS-177233)

### CODE REVIEW

**Comment by Artartur** [comment]
ci:test:object
ci:test:relevantci:test:sf

**Comment by Artartur** [comment]
ci:stop

**Comment by Artartur** [comment]
ci:test:sf

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by Artartur** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java [hunk 1]
```diff
@@ -132,24 +132,19 @@ public enum DDMExpressionOperator {
...
 				"independent-relationship",
 			"or"),
-		PLUS(
-			"field_name + field_name2",
-			"add-numeric-fields-to-create-an-expression", "plus"),
-		TIMES(
-			"field_name * field_name2",
-			"multiply-numeric-fields-to-create-an-expression", "times");
+		PLUS("+", "add-numeric-fields-to-create-an-expression", "plus"),
+		TIMES("*", "multiply-numeric-fields-to-create-an-expression", "times");
```
Comment (carolmariaabb) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java:147]: @Artartur, this fix does not seem to be the correct one for the described issue.

The content `field_name + field_name2` helps the user to understand how to build the expression with the `PLUS` operator, for example. Same thing for the other operators.

The most appropriate fix would be to handle the error that is thrown when accessing the custom object entries page.

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**